nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestListenRELP.java

```diff
@@ -91,18 +92,26 @@ public class TestListenRELP {
     @Mock
     private RestrictedSSLContextService sslContextService;
 
+    private AutoCloseable closeable;
+
     private RELPEncoder encoder;
 
     private TestRunner runner;
 
-    @Before
+    @BeforeEach
     public void setup() {
         encoder = new RELPEncoder(CHARSET);
         ListenRELP mockRELP = new MockListenRELP();
         runner = TestRunners.newTestRunner(mockRELP);
+        closeable = MockitoAnnotations.openMocks(this);
+    }
+
+    @AfterEach
+    public void cleanup() throws Exception {
+        closeable.close();
     }
```

Comment:
This should be unnecessary for a mocked object.

Reply:
Removing it